documentation/slate/source/includes/objects.md

```diff
@@ -658,15 +658,15 @@ A list of object that maps each lines with its associated stop area equipments r
     ...
 ]
 ```
-A list of objects that maps equipments details for each stop area.
+A list of objects that maps equipment details for each stop area.
```

Comment:
I do not agree with this change. There might be several equipments in a stop area
```suggestion
A list of objects that maps equipments details for each stop area.
```

Reply:
I tend to agree with mbounabi.